docker-compose.yaml

```
@@ -34,9 +34,17 @@ services:
       ST_SECRET_KEY:
       ST_DATABASE_URI: "mysql+pymysql://${MYSQL_USER}:${MYSQL_PASSWORD}@mysql/${MYSQL_DATABASE}"
       MINIO_ENDPOINT: minio:9000
-      MINIO_ACCESS_KEY:
-      MINIO_SECRET_KEY:
-      MINIO_REGION_NAME:
+      MINIO_ACCESS_KEY: "${MINIO_ACCESS_KEY}"
+      MINIO_SECRET_KEY: "${MINIO_SECRET_KEY}"
+      MINIO_REGION_NAME: "${MINIO_REGION_NAME}"
+      ENABLE_OIDC: "${ENABLE_OIDC:-}"
+      OIDC_CLIENT_ID: "${OIDC_CLIENT_ID}"
+      OIDC_CLIENT_SECRET: "${OIDC_CLIENT_SECRET}"
+      OIDC_WELL_KNOWN: "${OIDC_WELL_KNOWN}"
+      OIDC_PROVIDER: "${OIDC_PROVIDER}"
+      SENDGRID_API_KEY: "${SENDGRID_API_KEY}"
+      SENDGRID_UNSUBSCRIBE_GROUP_ID: "${SENDGRID_UNSUBSCRIBE_GROUP_ID}"
+      SENDGRID_EMAIL_TEMPLATE_ID: "${SENDGRID_EMAIL_TEMPLATE_ID}"
```

Looks like this merge went the wrong way. The result should be
```suggestion
      MINIO_ACCESS_KEY:
      MINIO_SECRET_KEY:
      MINIO_REGION_NAME: 
      SENDGRID_API_KEY:
      SENDGRID_UNSUBSCRIBE_GROUP_ID:
      SENDGRID_EMAIL_TEMPLATE_ID:
```